lib/vagrant-managed-servers/action/sync_folders.rb

```diff
@@ -62,6 +62,10 @@ def call(env)
                                 :hostpath => hostpath,
                                 :guestpath => guestpath))
 
+            # Escape backlashes in case the user is trying to authenticate using
+            # Active Directory domain credentials (e.g. "domain\username")
+            ssh_info.gsub!("\\", "\\\\\\\\")
```

Comment:
that looks totally weird, but in fact I did something similar [in some other place](https://github.com/tknerr/vagrant-workflow-tests/blob/master/spec/helpers.rb#L29)... ;-)